src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.heading {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
```

Comment:
that seems wrong to me adding size/positioning to title. Do you really need it? What is it for?

Reply:
I'm just making h1 visually hidden using [this](https://gist.github.com/luxplanjay/6f6936b1bba7afe006fa9317b07b6045) code